docs/source/changelog.rst

```diff
@@ -4,6 +4,94 @@
 Changes in nbconvert
 ====================
 
+5.6
+---
+
+The following 24 authors and reviewers contributed 224 commits -- Thank you all!
+* 00Kai0
+* Aidan Feldman
+* Alex Rudy
+* Alexander Kapshuna
+* Alexander Rudy
+* amniskin
+* Carol Willing
+* Dustin H
+* Hsiaoming Yang
+* imtsuki
+* Jessica B. Hamrick
+* KrokodileDandy
+* Kunal Marwaha
+* Matthew Seal
+* Matthias Geier
+* Miro Hrončok
+* M Pacer
+* Nils Japke
+* njapke
+* Sebastian Führ
+* Sylvain Corlay
+* Tyler Makaro
+* Valery M
+* Wayne Witzel
+
+The full list of changes they made can be seen `on GitHub <<https://github.com/jupyter/nbconvert/issues?q=milestone%3A5.6+>`__
+
+Significant Changes
+~~~~~~~~~~~~~~~~~~~
+
+Parallel NBConvert
+++++++++++++++++++
+NBConvert --execute can now be run in parallel via threads, multiprocessing, or async patterns! This means you can now parallelize nbconvert via a bash loop, or a python concurrency pattern and it should be able to execute those notebooks in parallel.
+
+Not all kernels are safe to execute concurrently. The ipython kernel as of version 1.5.2, for python 3, should now be safe to run concurrently. But the python 2 version for ipython sometimes fails with a socket bind exception. Other kernels were not tested heavily.
+
+Issues for nbconvert can be viewed here: :ghpull:`1018`:, and :ghpull:`1017`:
+
+We'll keep an eye for issues related to this new capability and try to quickly patch any discovered issues post release. The improvement required touching three projects with separate releases, so if you do find an issue try upgrading dependencies and listing your dependencies for your environment when reporting.
+
+Execute Loop Rewrite
+++++++++++++++++++++
+The execution loop responsible for monitoring kernel messages until cell execution is completed was rewritten. This removes an error where kernel messages could be dropped if too many were posted too quickly. Furthermore the change means that messages are not buffered and can be logged immediately rather than waiting for the cell to terminate.
```

Comment:
```suggestion
This release completely rewrote the execution loop responsible for monitoring kernel messages until cell execution is completed. This removes an error where kernel messages could be dropped if too many were posted too quickly. Furthermore, the change means that messages are not buffered. Now, messages can be logged immediately rather than waiting for the cell to terminate.
```